app/lib/backend_api_logic/routing_policy.rb

```diff
@@ -64,6 +64,13 @@ def path_to_regex
             "/#{path}/.*|/#{path}/?"
           end
         end
+
+        def replace_path
+          return {} if path.blank?
+
+          private_base_path = URI.parse(private_endpoint).path.to_s.sub(/^\//, '').sub(/\/$/, '')
```

Comment:
You might want to use https://github.com/3scale/porta/blob/83a66a93c9bf04a3ae383327e111c324bd1b8096/app/lib/string_utils/strip_slash.rb#L8

Reply:
Hmmm. It should not be `gsub`.